TI-reports/2025/2025-Q3-SCP-WG.md

```diff
@@ -0,0 +1,118 @@
+# 2025 Q3 Securing Critical Projects Working Group
+
+## Overview
+
+Sub-projects continue to run. Malicious Packages continues to
+attract interest, and is splitting from Package Analysis. Attendance is low.
+
+[2025-Q2 Update](2025-Q2-SCP-WG.md)
+
+## Identifying Critical Projects
+
+### Purpose
+
+Open Source Software has long suffered from a "tragedy of the commons"
+problem. Organizations large and small make use of OSS every day, but many
+projects are struggling for the time, resources and attention they need.
+
+This is a resource allocation problem - and we can help solve it together. We
+need ways to connect critical projects we all rely on with organizations that
+can provide them with support.
+
+[MVSR Link](https://github.com/ossf/wg-securing-critical-projects/blob/main/MVSR.md)
+
+### Current Status
+
+- Leveraging new, more comprehensive data sets
+    - [Census III](https://www.linuxfoundation.org/research/census-iii)
+    - [Ecosyste.ms](https://ecosyste.ms/) data
+
+### Up Next
+
+- Now that we have a good idea of critical projects, OSTIF is proposing a Managed Audit Program to go out and do the following to projects identified as critical:
```

Comment:
Interesting! Where would the funding come from for this work? I know the OpenSSF TI funding has been used for security audits of OpenSSF TIs (which makes sense), but I don't think we'd use TI funding for auditing external projects (or if we did, due to the budget it wouldn't be more than a few per year)? Maybe this would be more in Alpha Omega's wheelhouse?

Reply:
In my opinion this has been a long time issue: where does this WG stop and where does A.O. start?